web-app/helpers/certificate-designs.ts

```diff
@@ -28,29 +27,105 @@ function addText(canvas: Canvas, text: string, font: string, fillStyle: string,
   context.fillText(text, leftPos, rightPos);
 }
 
+// split long text into shorter lines
+function wrapText(canvas: Canvas, text: string, x: number, y: number, maxWidth: number, lineHeight: number, font: string, fillStyle: string) {
+  const context = getBaseContext(canvas);
+  const words = text.split(' ');
+  let line = '';
+  let y2 = y;
+  for (let n = 0; n < words.length; n += 1) {
+    const testLine = `${line} ${words[n]}`;
+    const metrics = context.measureText(testLine); // Check the width of the text, before writing it on the canvas
+    const testWidth = metrics.width;
+
+    // doesn't really understand the condition
+    // ask Ryan about it
+    if (testWidth > maxWidth && n > 6) {
+      context.fillStyle = fillStyle;
+      context.font = font;
+      // The x-axis coordinate of the point at which to begin drawing the text, in pixels.
+      // The y-axis coordinate of the baseline on which to begin drawing the text, in pixels.
+      context.fillText(line, x, y2);
+      line = `${words[n]}`;
+
+      y2 += lineHeight;
+    } else {
+      line = testLine.trim();
+    }
+  }
+  context.fillText(line, x, y2);
+}
+
+// eslint-disable-next-line max-lines-per-function
 export async function populateDeveloperCert(canvas: Canvas, details: any) {
-  // TODO
   console.log('populateDeveloperCert', { details });
```

Comment:
You need to update this function name (including the string in the console log here), right? Because it's no longer just about the Developer program.

Reply:
done